hw/ip/hmac/dv/cov/hmac_cov_excl.el

```diff
@@ -33,6 +33,35 @@ INSTANCE: tb.dut.u_reg.u_socket.err_resp
 ANNOTATION: "[UNR] Error response is captured in tlul_interface level, no error will land here in hmac"
 Block 4 "3222429784" "err_req_pending <= 1'b1;"
 Block 16 "2498388664" "err_rsp_pending <= 1'b1;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_wipe_secret
+ANNOTATION: "[UNR] qe is a constant, the coverage tool did not recognize it"
+Block 3 "0" "assign qe = we;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key0
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key1
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key2
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key3
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key4
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key5
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key6
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key7
+Block 1 "0" "assign qs = d;"
```

Comment:
Hi @eunchan , these are new exclusions that we have not reviewed. They are due to the coverage tool limitations, when you assign a constant, the line coverage will show uncovered. 
For example for `key0` in `prim_subreg_ext.sv`:
```
  assign qs = d;  // this is shown as a coverage hole because qs is never changed.
  assign q = wd;
  assign qe = we;
  assign qre = re;
```
I waived them. Please let me know if you have any concerns.